src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -39,16 +42,22 @@ public ServiceManagerBuilder WithCallingAssembly()
         /// <returns>The instance of the <see cref="IServiceManager"/>.</returns>
         public IServiceManager Build()
         {
-            _options.ValidateOptions();
-
+            _services.PostConfigure<ServiceManagerOptions>(o => o.ValidateOptions());
```

Comment:
reuse `AddSignalRServiceManager`?

Reply:
Yes, actually in the final version it reuses it...